packages/nimble-components/src/toolbar/tests/toolbar.mdx

```diff
@@ -0,0 +1,35 @@
+import { Canvas, Meta, Title } from '@storybook/blocks';
+import { NimbleToolbar } from './toolbar.react';
+import * as toolbarStories from './toolbar.stories';
+
+<Meta of={toolbarStories} />
+<Title of={toolbarStories} />
+
+Per [W3C](https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/) - A toolbar is a container
+for grouping a set of controls, such as buttons, menubuttons, or checkboxes.
+
+When a set of controls is visually presented as a group, the toolbar role can be used to communicate the
+presence and purpose of the grouping to screen reader users. Grouping controls into toolbars can also be
+an effective way of reducing the number of tab stops in the keyboard interface.
+
+To override the toolbar's background color, style the `positioning-region` part:
+
+```css
+nimble-toolbar::part(positioning-region) {
+    background: red;
+}
+```
```

Comment:
Was wondering if we could just remove this but looks like there are [at least several usages](https://dev.azure.com/ni/_search?action=contents&text=nimble-toolbar%3A%3Apart(positioning-region)&type=code) (this search may not catch all).

The bummer is not all of them are using theme aware tokens. Some are explicitly using a base token :/
Probably means we should have a few appearance modes as options to understand the usages and make this more consistent. Created this tech debt for it: https://github.com/ni/nimble/issues/1800